From 6e45679085080bb0038077d9067668d5f5114392 Mon Sep 17 00:00:00 2001
From: DigitalDreamtime <clive.messer@digitaldreamtime.co.uk>
Date: Mon, 4 Jul 2016 00:00:00 +0100
Subject: [PATCH 06/19] 384k: Dion Audio LOCO: Support 384k max sample rate

Add support for 384k max sample rate to dionaudio-loco overlay.

Signed-off-by: DigitalDreamtime <clive.messer@digitaldreamtime.co.uk>
---
 arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts b/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts
index 3930f41..f8a9eba 100644
--- a/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts
@@ -24,6 +24,7 @@
 				#sound-dai-cells = <0>;
 				compatible = "ti,pcm5102a";
 				status = "okay";
+				pcm5102a,384k;
 			};
 		};
 	};
-- 
2.7.4

